searchlib/src/vespa/searchlib/engine/proto_converter.cpp

```diff
@@ -58,6 +58,7 @@ ProtoConverter::search_request_from_proto(const ProtoSearchRequest &proto, Searc
     request.maxhits = proto.hits();
     request.setTimeout(1ms * proto.timeout());
     request.setTraceLevel(proto.trace_level());
+    request.trace().setProfileDepth(proto.profile_depth());
```

Comment:
test should be extended to verify conversion of this field

Reply:
Done